core/src/main/resources/com/squarespace/template/plugins/platform/variants-select.html

```diff
@@ -12,7 +12,8 @@
         <option value="">{selectText|message variantName:name}</option>
         {.repeated section values}<option value="{@|htmltag}">{@|htmltag}</option>{.end}
       </select>
-    </div>
+    </div>{.repeated section values}
```

Comment:
The {.repeated ...} part could use its own line for readability

Reply:
see my comments under george's feedback for why this has to be on the same line